src/shared/configuration.ts

```diff
@@ -4,8 +4,9 @@ export const configuration = {
   // __dirname is equivalent to the installation path of dist/main.umd.js.
   // This way the commands can be executed locally as well with "npm run self:install/check/update".
   moduleBasePath: path.join(__dirname, "..", "src"),
-  docFolderName: "doc",
+  outdatedDocFolderName: "doc",
+  docsFolderName: "docs",
   rootFolderName: "root",
-  docBackupFolderName: "_doc",
+  docsBackupFolderName: "_docs",
```

Comment:
This is not super important, but this would be the perfect chance to choose a different backup folder name if we wanted that.

I'm not sure if it's really better, but personally I'd prefer a more verbose name like `docs_backup` or even `docs_backup_please_rename`. The sillier the name the less likely it is that something else is using that folder name. I can think of at least 2 frameworks that want you to put underscores at the start of folders to carry some meaning about routing or import rules. The chances are still low, but if we can change it for free… 🤷🏻 

This will definitely not keep me from approving the PR!